[2021-10-16 13:05:31,043] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T12:04:00+00:00 [queued]>
[2021-10-16 13:05:31,206] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T12:04:00+00:00 [queued]>
[2021-10-16 13:05:31,207] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:31,207] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:31,207] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:31,506] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 12:04:00+00:00
[2021-10-16 13:05:31,529] {standard_task_runner.py:52} INFO - Started process 823 to run task
[2021-10-16 13:05:31,577] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T12:04:00+00:00', '--job-id', '434', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmp6c8eso88', '--error-file', '/tmp/tmpde1q_irk']
[2021-10-16 13:05:31,578] {standard_task_runner.py:77} INFO - Job 434: Subtask get_gross_year_1
[2021-10-16 13:05:31,900] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T12:04:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:32,224] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T12:04:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T12:04:00+00:00
[2021-10-16 13:05:32,267] {python.py:152} INFO - Done. Returned value was: ['$160.31 ', '2008\n']
[2021-10-16 13:05:32,629] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T120400, start_date=20211016T130531, end_date=20211016T130532
[2021-10-16 13:05:32,710] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:05:32,996] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
